packages/@storefront/sort/CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [patch]
+### Changed
+- SF-1256: Sort component will now whenever it recieves new props.
```

Comment:
"will now" what?

Reply:
Whoops, looks like we missed this. Will update.